class/defaults.yml

```diff
@@ -2,3 +2,44 @@ parameters:
   vcluster:
     =_metadata: {}
     namespace: syn-vcluster
+
+    base:
+      service_cidr: null
```

Comment:
Why don't we set a usable default for the cluster service cidr? Or is this the host cluster's service cidr?

Reply:
It's the host clusters service cidr. I'll rename the key.